backends/bmv2/run-bmv2-test.py

```diff
@@ -181,9 +189,21 @@ def run_model(options, tmpdir, jsonfile):
     result = bmv2.checkOutputs()
     return result
 
+def make_make_targets(options):
```

Comment:
It's a lot of machinery to support one test case, why don't you put all these commands into a shell script and just invoke the shell script from cmake?

Reply:
I thought it would be more reusable this way.